### PR TITLE
Ott 1418

### PR DESCRIPTION
**Pull request by warbhe-amit**
Updated messageHandler with the return value for the post trigger events

### CODE REVIEW

File: server/src/messageHandler.mjs [hunk 1]
```diff
@@ -35,7 +36,7 @@ function emit(id, result, msg, ws) {
       result: result
     };
     const eventMessage = JSON.stringify(oEventMessage);
-    ws.send(eventMessage);
+    ws.send(eventMessage);
```
Comment (michaelcfine) [inline server/src/messageHandler.mjs:39]: Looks like some stray spaces / tab at the end of this line.

**Review by michaelcfine — COMMENTED** [review]
Looks great! Just some stray whitespace at the end of one line; otherwise ready to approve and then you can merge.

File: server/src/messageHandler.mjs (hunk 1 above)
Comment (warbhe-amit) [reply server/src/messageHandler.mjs]: removed stray white space at line # 39

**Review by michaelcfine — APPROVED** [review]
👍 Looks good! Merge away, unless you want me to.

**Pull request merged**